drivers/interrupt_controller/intc_gicv3.c

```diff
@@ -248,8 +254,10 @@ static void gicv3_dist_init(void)
 		/* Clear pending */
 		sys_write32(BIT_MASK(GIC_NUM_INTR_PER_REG),
 			    ICPENDR(base, idx));
-		/* All SPIs are G1S and owned by Zephyr */
-		sys_write32(0, IGROUPR(base, idx));
+		if (IS_ENABLED(CONFIG_ARMV8_A_NS))
```

Comment:
`CONFIG_ARMV8_A_NS` is a boolean and is guarding a pretty trivial `sys_write32` so why don't you just use an `ifdef`?

Reply:
fix in next version.